pandas/core/base.py

```diff
@@ -737,11 +737,17 @@ def item(self):
     @property
     def data(self):
         """ return the data pointer of the underlying data """
+        warnings.warn("Series/Index.data is deprecated and will be "
```

Comment:
Might be nice to make the warning messages a bit more dynamic with something like: 
```python
"{obj}.data is deprecated...".format(obj=type(self).__name__)
```

Reply:
Yes, thank you (somehow I was the docstring mindset thinking we needed to adapt this upon class definition, but of course this can be done dynamically on run time :-))